src/static-build/utils.tsx

```diff
@@ -56,3 +56,16 @@ export function escapeStyleScriptContent(str: string): string {
     .replace(/<style/g, '<\\style')
     .replace(/<\/style/g, '<\\/style');
 }
+
+/**
+ * Removes empty lines from start and end, and removes the initial indent from
```

Comment:
I probably would have used `dedent` from npm to avoid dealing with getting regexes right, but maybe it's not worth it for something still fairly small :)

Reply:
Done! I wasn't aware of dedent.